src/diffusers/models/model_loading_utils.py

```diff
@@ -221,7 +221,7 @@ def _fetch_index_file(
                 local_files_only=local_files_only,
                 token=token,
                 revision=revision,
-                subfolder=subfolder,
+                subfolder=None,
```

Comment:
Why should this be `None`, though?

Reply:
That is because `subfolder` is already determined here:
https://github.com/huggingface/diffusers/blob/7833ed957bcb4bd8c9ed0a9a8a172513bf896e51/src/diffusers/models/model_loading_utils.py#L209